### ISSUE TITLE
REQUEST: compatibility Korppis-Spearfishing-1.0.2

### ISSUE THREAD

**Issue by BETLOG** [shudnal/HarpoonExtended#3]
Please consider adding some form of configuration/compatibility with Korppis-Spearfishing-1.0.2
Currently if i harpoon a fish it dies, and isnt tethered, so i still have to swim out to get it.
I have also requested that they add[ compatibility with HarpoonExtended](https://github.com/karkkant/valheim-spearfishing/issues/9), because I suspect this is the correct way to do it, but idk.

**Comment by shudnal** [comment]
I've checked the code of spearfishing and I can't see a way to provide compatibility from my side. The mod is very simple and only check for is projectile a regular harpoon indirectly which is causing issues with HarpoonExtended. Any config changes on HarpoonExtended side will not work since Spearfishing doesn't check for it. It works simply if fish is hit by projectile - destroy fish and spawn loot.
I've commented on the issue you mentioned.
